pages/index.html

```diff
@@ -20,12 +20,12 @@
 <!-- Add HTML markup for example here -->
 </div>
 
-<div class="grid-box">
-  <div class="grid-item width-one-half annotation">
+<div class="usa-grid-box">
+  <div class="width-one-half annotation">
     <h3>Use</h3>
     <p>This is where you'll find basic guidelines about the usage of a component. For example, a minimal palette, clear hierarchy, good information design, and ample whitespace will ensure a voice of authority and expertise in communication.</p>
   </div>
-  <div class="grid-item width-one-half annotation">
+  <div class="width-one-half annotation">
```

Comment:
@msecret Open to adding or waiting too. Maybe it'll be more consistent to just add it tho.